src/services/symbolication.rs

```diff
@@ -312,6 +312,7 @@ impl ModuleListBuilder {
         for thread in minidump_process_state.threads() {
             for frame_pair in thread.frames().windows(2) {
                 if let [prev_frame, next_frame] = frame_pair {
+                    // TODO(flub): This never marks the first frame!
```

Comment:
what do you mean by this?

Reply:
whoops, this is not supposed to be part of this PR... but i suspect there is a bug here and need to check this.